src/TwigYard/Middleware/Form/Handler/ApiHandler.php

```diff
@@ -224,10 +263,15 @@ private function validateConfigData(): void
                         self::CONFIG_DATA_FORMAT_FLOAT,
                         self::CONFIG_DATA_FORMAT_BOOL,
                     ];
+                    $supportedTypesExt = array_merge(
+                        $supportedTypes,
+                        [self::CONFIG_DATA_FORMAT_DATETIME]
+                    );
 
                     if (
-                        array_key_exists(self::CONFIG_DATA_FORMAT, $this->config)
-                        && !in_array($this->config[self::CONFIG_DATA_FORMAT], $supportedTypes)
+                        array_key_exists(self::CONFIG_DATA_FORMAT, $dataValue)
```

Comment:
@mbohal This if-block was wrong. I replaced `$this->config` with `$dataValue` to fix it. It was not critical fix, because it only checks if config is valid, but still it was bug.